ophyd/tests/test_sim.py

```diff
@@ -0,0 +1,37 @@
+from ophyd.sim import SynGauss, Syn2DGauss, SynAxis
+import numpy as np
+
+
+def test_random_state_gauss1d():
+    """With given random state, the output value should stay the same.
+    Test performs on 1D gaussian.
+    """
+    dlist=  []
+    motor = SynAxis(name='motor')
+    for i in range(2):
+        s = np.random.RandomState(0)
+        noisy_det = SynGauss('noisy_det', motor, 'motor', center=0, Imax=1,
+                             noise='uniform', sigma=1, noise_multiplier=0.1,
+                             random_state=s)
+        noisy_det.trigger()
+        d = noisy_det.read()['noisy_det']['value']
+        dlist.append(d)
+    assert dlist[0] == dlist[1]
+
```

Comment:
I would try a test without random_state and run
```python
for i in range(2):
    np.random.seed(0)
    noisy_det = SynGauss(...)
```
so basically try it without `random_state` flag.

Reply:
done.

Reply:
Good suggestion, @jrmlhermitte. Looks good, @licode.